src/scene/graphics/shared/Graphics.ts

```diff
@@ -22,14 +26,10 @@ import type { ConvertedFillStyle, ConvertedStrokeStyle, FillStyle, FillStyleInpu
  * @see {@link scene.Graphics}
  * @memberof scene
  */
-export interface GraphicsOptions extends Partial<ContainerOptions<GraphicsView>>
+export interface GraphicsOptions extends ContainerOptions
 {
     /** The GraphicsContext to use, useful for reuse and optimisation */
     context?: GraphicsContext;
-    /** The fill style to use */
-    fillStyle?: FillStyleInputs;
-    /** The stroke style to use */
-    strokeStyle?: FillStyleInputs;
```

Comment:
why have these been removed?

we have getter/setters for these on the Graphics class os they should be part of the constructor options?